### PR TITLE
Map Overhaul

### PR DESCRIPTION
**Pull request by Stutternov**
## About The Pull Request
Big map overhaul. Posted tons of images of this in the discord, I will give some basic highlights here. 
- Redoes the colony from its current 3Zs to a _new_ 3Z map, inspired by our olllld map from 2+ years back
- Z1, the lowest Z-level, is now mostly lower-office rooms, Genetics, Xeno, AI-sat, and otherwise is **ALL MAINTENCE** and various dungeons for the public to run; providing some dungeons players can do that are not prospectors for some OK and rare loot.
- Z2, the mid-layer, is the main operations layer. Containing Medical, Science, most of the Church, the Bar, dorms, 2 shops, Lonestar and Guild (who now share a lobby) and some basic colony maintenance; such as the old pool, basketball court, etc - which are left in disrepair. These areas are low-risk to go to and fix up, or to get some low-grade salvage from.
- Z3, the top layer, has remained the least changed - but still sporting many changes. Prospectors are moved and their ship is now outside - plus they have a proper shop now, Marshals and Blackshield have been merged but their armories have stayed separate (Marshal one and shop is on Z-3, BS one is behind command on Z-2), Church commons have remained up there though condensed a bit.

Pictures are in map-dev. **Ready for testing.**

image

## Changelog
:cl:
adds: complete map overhaul
removes: my sanity
/:cl:

### CODE REVIEW

**Comment by ThePainkiller** [comment]
I for one, really appreciate the fact medical is in the middle z-level as that makes it far easier and faster to reach any of the other two z-levels for rescue operations

**Comment by Stutternov** [comment]
> I for one, really appreciate the fact medical is in the middle z-level as that makes it far easier and faster to reach any of the other two z-levels for rescue operations

I'll be honest that I do not know too much about Medical's issues - such as traffic-issues between high-pop areas and low-pop areas.

If you have an input on moving any rooms around, please do tell me btw. I've tried to make storage and offices, aka low-traffic areas, on the lower to encourage it more as a 'secluded space for roleplay'. While the top-side, the operations deck, is much more high-traffic. BUT - I'm unsure if it would flow the same, better, or worse than current.

**Comment by benj8560** [comment]
CBO gets their own ~~ERP room~~ bathroom but no one else does? HmmmmmMMMMmmm. I say all or nothing medicals got their own entire showers.

**Comment by Stutternov** [comment]
Map is basically finished and ready for testing. Fixed a few wire issues, expanded Guild back a bit. I'll finish the space-map at a later point because I need an event for it.

File: maps/SpaceFortress/spacefortress.dm [hunk 1]
```diff
@@ -1,5 +1,6 @@
 //#include "map/spacefortress.dmm"
 //#include "map/blackstone_temple.dmm"
+#include "map/space_dungeons.dmm"
```
Comment (Trilbyspaceclone) [inline maps/SpaceFortress/spacefortress.dm:3]:
```suggestion
//#include "map/space_dungeons.dmm"
```

**Comment by WatermelonsEverywhere** [comment]
Are the corpses in maintenance supposed to show up on sensors? The one in the deathclaw cave did when I locally tested.

**Comment by Stutternov** [comment]
> Are the corpses in maintenance supposed to show up on sensors? The one in the deathclaw cave did when I locally tested.

Agony. Not intended, will have to look at code-changing that.

**Pull request merged**